formula-android/src/main/java/com/instacart/formula/android/FeatureView.kt

```diff
@@ -0,0 +1,30 @@
+package com.instacart.formula.android
+
+import android.view.View
+import com.instacart.formula.Cancelable
+import com.instacart.formula.fragment.FragmentLifecycleCallback
+import com.instacart.formula.fragment.FormulaFragment
+import io.reactivex.rxjava3.core.Observable
+
+/**
+ * Feature view provides [FormulaFragment] with the root Android view which will be returned as
+ * part of [FormulaFragment.onCreateView] and the logic to bind the state observable to the
+ * rendering. Formula fragment uses [ViewFactory.create] to instantiate [FeatureView].
+ *
+ * Use [ViewFactory.fromLayout] and [LayoutViewFactory] to define a [ViewFactory] which can create
+ * [FeatureView].
+ *
+ * @param view The root Android view.
+ * @param bind A bind function connects state observable to the view rendering.
+ * @param lifecycleCallbacks Optional lifecycle callbacks if you need to know the Fragment state.
+ */
+class FeatureView<RenderModel>(
```

Comment:
Do we still want to use the term `View` in so many places? Maybe `Screen` instead, `ScreenFactory`, `FeatureScreen`, etc? I am just wondering if we want to move away from `View` since Compose doesn't use Android Views, and though they are unrelated, they might be associated. Maybe even `FeatureUI`, `UIFactory`, etc? Naming is hard, so I understand if you disagree.